glomentorprogram/src/components/GenreSelect/GenreSelect.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import './GenreSelect.css';
+
+const GenreSelect = ({ GenreList }) => {
```

Comment:
@bcvitanusic the requirement for this implementation is to pass 3 props: genreslist, current selected and callback and you pass only one. Please review the requirements to fix it

Reply:
fixed, passed 3 props